mantidproject/mantid
# Fix error reporter inside MantidPlot

**Description of work.**

The recent macOS package merge broke the error reporter inside MantidPlot on Linux systems either without the workbench built or running from a package. This fixes the issues.

**Report to:** [nobody]. 

**To test:**

On a linux system pass these additional options to cmake:
`-DENABLE_CPACK=ON -DBUILD_WORKBENCH=ON -DPACKAGE_WORKBENCH=ON` and run `ninja package`. Install the new package and verify that the error reporter starts when a terminate/continue error is generated.

*There is no associated issue.*

*This does not require release notes* because **it was broken in a dev cycle**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
